tests/test.dispensary.js

```diff
@@ -157,10 +200,26 @@ describe('Dispensary', function() {
       });
   });
 
+  it('should return cached libraries after first call to getLibraries', () => {
+    var dispensary = new Dispensary();
+
+    fsSpy.reset();
+
+    dispensary.getLibraries()
```

Comment:
It might result in things not being asserted as expected since the tests don't know it's async.